services/category.js

```diff
@@ -15,6 +16,16 @@ const categoryService = {
     return await Category.findById(id).lean().exec()
   },
 
+  addCategory: async (data) => {
+    let { name, categoryIcon } = data
+
+    if (name[0] === name[0].toLowerCase()) {
```

Comment:
do you need this check, if yes better move it to capitalizeFirstLetter function, and you will avoid using this check in multiple places

Reply:
@Tolik170 it has a sense to stay this condition cause we can avoid unnecessary iterations during slicing in capitalizing letter method

Reply:
agree with @mxrcury

Reply:
fixed